src/cf-expandables/src/cf-expandables.less

```diff
@@ -50,7 +45,7 @@
 
 .o-expandable {
     position: relative;
-    margin-bottom: unit( 30px / @base-font-size-px, em );
+    //margin-bottom: unit( 30px / @base-font-size-px, em );
```

Comment:
Can we delete this?

Reply:
Here and the other commented out margin-bottom bits.

Reply:
I left them as part of the discussion about breaking changes, but my intention is to delete them.